Pollo/ViewControllers/CardController+Extension.swift

```diff
@@ -260,7 +260,12 @@ extension CardController: SocketDelegate {
             delegate?.pollStarted(poll, userRole: userRole)
             return
         }
+
+        if pollsDateModel.polls.contains(where: { $0.isEqual(toDiffableObject: poll)}) {
+            pollsDateModel.polls.removeLast()
```

Comment:
why do we `removeLast()`, shouldn't we remove the index of the duplicate poll?

Reply:
i'm pretty sure the duplicate/active poll is always going to be the last poll in the list, but i guess it could be safer to remove at the index of the duplicate?

Reply:
fixed with a filter!